### DISCUSSION TITLE
Setting the maxDuration timeout in Remix causes a 404

### DISCUSSION THREAD

**Discussion by Klingefjord** [vercel/community#4244] [Bug Report]
### Page to Investigate

https://github.com/remix-run/remix/issues/7537

### Steps to Reproduce

See https://github.com/remix-run/remix/issues/7537

**Comment by felquis** [comment]
I've setup a new fresh remix project with remix v2, added these lines to `app/routes/test.tsx`

```tsx
export const config = {
  maxDuration: 30,
};

export async function loader() {
  const d = async () => new Promise((resolve) => setTimeout(resolve, 16000));

  await d();

  return { message: "Hello" };
}

export default function App() {
  return <h1>Loading..</h1>;
}
```

~~It consistently timeout at 15s~~
It doesn't return 404 or 500 on remix v2
I'm using a pro plan

**Reply by felquis** [reply]
After a while I notice it did work, it looks like `export const config` only works with remix v2 routing convention. I've updated my webapp routes, and it worked as expected.

**Comment by TooTallNate** [comment]
Can you provide a standalone repository that demonstrates the issue?
